static/js/util/util.js

```diff
@@ -24,3 +24,7 @@ export function createForm(url: string, payload: Object): HTMLFormElement {
 }
 
 export const isNilOrBlank = R.either(R.isNil, R.isEmpty);
+
+export const formatDollarAmount = (amount: ?number): string => (
+  amount ? `$${amount.toLocaleString()}` : ''
```

Comment:
I'm not sure we want to use `toLocaleString` without setting an explicit locale here. The `toLocaleString` method on `Number` also has some facilities for formatting currency values automatically, this will take a number value and format it to USD:

```js
let num = 42;
num.toLocaleString('en-US', {style: "currency", currency: "USD"});
// $42.00
```

If we don't want the decimal points in the price there's an option called `maximumFractionDigits` which can be set to 0.

I think we want to set `en-US` explicitly, otherwise we could get some strange stuff that might confuse the user. For instance, setting the locale to `de-DE` gives `42,00 $` for the same value.